core/tests/protractor_utils/editor.js

```diff
@@ -11,1367 +11,3 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Hi @hoangviet1993, shouldn't we completely delete this file if we do not require it?

Reply:
Yes we should! I am going to delete the file before the merge :+1:

Reply:
In general, it's OK to just go ahead and delete it -- no need to wait till before the merge. Also gives you some guarantee that your stuff is working fine without it.

Don't worry about merge conflicts. I don't actually see them happening in this case because people aren't really modifying protractor_utils/editor.js.

(I forgot to push on this earlier, but should have before merging #5139. We should never have duplicate code in the codebase.)